### PR TITLE
Career goals blog - Outreachy

### PR DESCRIPTION
**Pull request by deborahtrez**
About my Outreachy Internship with Mixxx: Career goals.
@ywwg @Holzhaus please, kindly review :)

### CODE REVIEW

**Comment by ywwg** [comment]
I read your blog post on your own site and I really liked it! I didn't see any typos so this LGTM

**Review by ywwg — APPROVED** [review]

**Comment by daschuer** [comment]
I would remove the comma before "and" in lists.
I don't think they are literally wrong in English but at least not recommended. (I German they would be wrong).

File: content/news/2021-02-22-career-goals.md [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+The goal is to make the [Mixxx](https://mixxx.org) software easier to use especially to the new user by making [the manual](https://manual.mixxx.org/2.2/en/) easy to understand and follow.
+I have also been making video tutorials for the [Mixxx](https://mixxx.org) YouTube channel; videos that will act as a guide for [Mixxx](https://mixxx.org) users to follow and make great mixes.
+
+#### My background
+
+I have always taken a keen interest in tech, but I didn’t get to fully explore it until I joined the university and enrolled in a Computer Engineering course.
+[High school](https://msmcnamagunga.ac.ug) provided a couple of opportunities in robotics and innovation, but with the limited resources and the vast number of students, the time was never enough to learn and master a new skill.
+But even with the limited resources, I am grateful to have had the chance to learn how to build lego prototypes and write simple programs for them.
+
+I started learning C+ and Python, as part of the engineering curriculum and this foundation propelled me into wanting to learn more about programming and seeing what else I could do with code.
```
Comment (daschuer) [inline content/news/2021-02-22-career-goals.md:24]:
```suggestion
I started learning C++ and Python, as part of the engineering curriculum and this foundation propelled me into wanting to learn more about programming and seeing what else I could do with code.
```

**Comment by Be-ing** [comment]
> I would remove the comma before "and" in lists.
I don't think they are literally wrong in English but at least not recommended. (I German they would be wrong).

This varies between and within English dialects: https://en.wikipedia.org/wiki/Serial_comma

**Comment by ywwg** [comment]
that's also known as the "oxford comma".  There's a lot of cases where it makes things more clear.  Some organizations have a style but for a personal post it's up to the writer to use one or the other as they choose

**Comment by Holzhaus** [comment]
Merge?

**Pull request merged**